src/openforms/config/admin.py

```diff
@@ -221,6 +221,20 @@ class RichTextColorAdmin(admin.ModelAdmin):
     ]
 
 
+@admin.register(MapTileLayer)
+class MapTileLayerAdmin(admin.ModelAdmin):
+    fields = [
+        "label",
+        "identifier",
+        "url",
+    ]
+    list_display = [
+        "label",
+        "identifier",
+        "url",
+    ]
```

Comment:
I tend to favour tuples rather than lists for these properties, because:

* tuples are immutable
* lists defined at the module level are mutable and people tend to try nasty things where they dynamically patch the admin class and add stuff (in tests) and then they get surprises 3 months later

In general I avoid using mutable datastructures at the module-level